packages/sakuli-legacy/src/context/sahi/sahi-api.interface.ts

```diff
@@ -14,4 +14,17 @@ export interface SahiApi extends ActionApi, AccessorApi, RelationApi, FetchApi,
      */
     _dynamicInclude(): Promise<void>;
 
-}
\ No newline at end of file
+    /**
+     * Sets the maximum time Sakuli will try to find an element on dom.
+     *
+     * @param timeOutMs - Timeout in milliseconds
+     */
+    _setFetchTimeout(timeOutMs: number): void;
+
+    /**
+     * Gets the maximum time Sakuli will try to find an element on dom.
+     *
+     * @param timeOutMs - Timeout in milliseconds
+     */
+    _getFetchTimeout(timeOutMs: number): void;
```

Comment:
```suggestion
    _getFetchTimeout(): number;
```
There is no parameter for `_getFetchTimeout()`

Reply:
Fixed in 8bc1bf2